src/v0/destinations/monday/util.js

```diff
@@ -154,14 +154,16 @@ const getColumnValue = (properties, columnName, key, board) => {
  */
 const mapColumnValues = (properties, columnToPropertyMapping, board) => {
   const columnValues = {};
-  columnToPropertyMapping.forEach((mapping) => {
-    columnValues[getColumnId(mapping.from, board)] = getColumnValue(
-      properties,
-      mapping.from,
-      mapping.to,
-      board,
-    );
-  });
+  if (isDefinedAndNotNull(columnToPropertyMapping)) {
```

Comment:
```suggestion
if (isDefinedAndNotNull(columnToPropertyMapping) && Array.isArray(columnToPropertyMapping)) {
```
Check if `columnToPropertyMapping` is an array or not.

Reply:
yes, `columnToPropertyMapping` is an array

Reply:
updated the condition